frontend/public/locales/en/unable-to-process-request.json

```diff
@@ -0,0 +1,14 @@
+{
+  "page-title": "Unable to process CDCP request",
+  "unable-to-process": "We are unable to process your Canadian Dental Care Plan (CDCP) request at this moment. This could be due to an incorrect input, a timeout, or a technical problem.",
+  "try-again": "To try again, you can:",
+  "disable-vpn": "disable your virtual private network (VPN), or browser or proxy server, if you use one; or",
+  "another-device": "try making the request on another device.",
+  "difficulties": "If you're still experiencing difficulties:",
+  "call-centre": "call the CDCP Specialized Call Center for assistance at 1-833-537-4342;",
```

Comment:
**Suggestion:** make the phone number no-wrap (there are examples of how to do this elsewhere in the application)

Reply:
[Fixed](https://github.com/DTS-STN/canadian-dental-care-plan/compare/851bd2280b6a2da882304895b50bcb5132c9c9ac..42daa04af648b2233f7846a8e00ac704edb32466)